### PR TITLE
Add release notes for box shadows

### LINKED ISSUE #1721: Write release notes for PR #15204: box shadow
https://github.com/bevyengine/bevy/pull/15204 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `/home/alice/Documents/bevy-website/release-content/0.15/release-notes/15204_box_shadow.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fixes #1721. These notes were developed from the PR description for https://github.com/bevyengine/bevy/pull/15204 :)

### CODE REVIEW

**Review by richchurcher — APPROVED** [review]

**Review by cart — COMMENTED** [review]
Text content is solid, but I think we should render a new "landscape" image. Portrait aspect ratios aren't great for the release notes.

The `box_shadow` example seems reasonable (and the blue is more color-pallet aligned), although it seems like there are some bugs: very hard edges for curves and what appears to be an x-axis scale on some of them). I reworked it slightly:

image

**Comment by cart** [comment]
New slightly better image that uses @aevyrie's suggestion of a 0.5 "subpixel blur" to clean up the edges on the first row:

image

**Comment by alice-i-cecile** [comment]
Looks nice! I take it you're in the process of a PR for that fix?

**Comment by ickshonpe** [comment]
Really like these adjustments, looks much better. I don't know about much graphics programming so I concentrated on getting the user API right. With the shader I just went with something simple in the hope that someone else would pick it up and improve the implementation later like this.

**Comment by ickshonpe** [comment]
> The `box_shadow` example seems reasonable (and the blue is more color-pallet aligned), although it seems like there are some bugs: very hard edges for curves and what appears to be an x-axis scale on some of them). I reworked it slightly:

When the box shadow PR was written, UI anti aliasing was bugged and edges on curves were sharper. If you didn't account for it already, maybe the subpixel blur needs to be turned off if AA is disabled?

**Review by ickshonpe — APPROVED** [review]
Sorry I remember now I said I'd write this myself. Really nice note, no complaints.

File: release-content/0.15/release-notes/15204_box_shadow.md [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+```rust
+#[derive(Component)]
+pub struct BoxShadow {
+    /// The shadow's color
+    pub color: Color,
+    /// Horizontal offset
+    pub x_offset: Val,
+    /// Vertical offset
+    pub y_offset: Val,
+    /// Horizontal difference in size from the occluding uninode
```
Comment (ickshonpe) [inline release-content/0.15/release-notes/15204_box_shadow.md:24]: Just noticed this is wrong. The original draft of the PR had separate horizontal and vertical spread values but that would have needed some sort of elliptical border radius support so it was removed but I must have missed this comment in the PR description. From the doc comments for the struct:

```suggestion
    /// How much the shadow should spread outward.
    ///
    /// Negative values will make the shadow shrink inwards.
    /// Percentage values are based on the width of the UI node.
```

**Review by ickshonpe — CHANGES_REQUESTED** [review]

**Comment by cart** [comment]
> Looks nice! I take it you're in the process of a PR for that fix?

This was all done in "userspace" by setting the box shadow blur value to `0.5`. We did discuss clamping values less than 0.5 to 0.5 automatically, but the blur does dirty up clean edges. In the short term I think the fix is to add docs about blur values of 0.5 cleaning up jagged curves. Long term we should consider sorting out a way to automatically anti-alias curves without dirtying up clean vertical and horizontal edges.

**Review by cart — APPROVED** [review]

**Pull request merged**